SimMuon/RPCDigitizer/src/RPCSimSetUp.cc

```diff
@@ -99,6 +99,8 @@ void RPCSimSetUp::setRPCSetUp(const std::vector<RPCStripNoises::NoiseItem>& vnoi
   }
 }
 
+// Previous version with hardcoded limit of max 96 strips per roll
+/*
```

Comment:
the old code should be removed or left available with configuration switch.
Large commented out blocks just to keep for historical reasons is not a  good practice